### PR TITLE
Modified info.json

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
Added missing required fields

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -1,15 +1,60 @@
...
     "label": "Multi Tenancy",
-    "description": "FortiSOAR Solution Pack for Multi Tenancy Use Cases",
+    "dependencies": [
+        {
+            "name": "fortiSOAREssentials",
+            "type": "solutionpack",
+            "label": "FortiSOAR Essentials"
+        },
+        {
+            "name": "socSimulator",
```
Comment (cs-suyog-jinturkar) [inline info.json:14]: Why MSSP is dependent on SoCSim?

Comment (cs-sujeet-ahirrao) [reply info.json:14]: if we want to execute any use-case then for that we have to create the record for same and hence we required soc simulator SP

Comment (cs-suyog-jinturkar) [reply info.json:14]: MSSP/MTT is a framework SP and not a scenario pack. Please correct me if i am wrong here?

Comment (cs-suyog-jinturkar) [reply info.json:14]: For now, we will keep this dependency but lets review the SP content and figure out why we need SoC Sim here.

**Pull request merged**